content/posts/2022-01-03-scm-manager-plugin-development/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: SCM-Manager - Recap 2021
+date: 2022-01-03T10:00:00+0000
+author: Eduard Heimbuch
+categories:
+- scm-manager
+- release
+keywords:
+- recap
+- plugin
+- development
+- guide
+- tutorial
+---
+
+Hey SCM-Manager Community,
+
+It's the first monday of the new year, and we just started working on the next major features to improve your SCM-Manager experience.
+But before going forward we wanted to do a little recap and share our highlights regarding SCM-Manager of 2021.
+
+## Recap
+We had 16 minor releases in the last year including these big features/changes:
+- Repository Import and Export for Git, HG and Subversion
+- Frontend rebuild with react-query which dramatically improved the performance
+- Metrics for SCM-Manager which can be collected with the related plugins as [JSON](https://scm-manager.org/plugins/scm-metrics-json-plugin/) or [PromQL](https://scm-manager.org/plugins/scm-metrics-prometheus-plugin/)
+- Global Notification System
+- Repository Mirroring with the [Repository Mirror Plugin](https://scm-manager.org/plugins/scm-repository-mirror-plugin/)
+- Global search for most entities in SCM-Manager
+- Rebuild of the repository overview
+- High Contrast Mode and improved accessibility in general
+- Introduction of the Cloudogu Plugin Center (with more features to come)
+
+Besides, the core changes we also added some helpful plugins which can lift your SCM-Manager workflows into a new level.
+We cannot explain all plugins here, but we strongly recommend having a look for the following:
+
+- [Gotenberg Plugin](https://scm-manager.org/plugins/scm-gotenberg-plugin/) - Renders office files like Word or Excel inside your SCM-Manager file view
+- [Landingpage Plugin](https://scm-manager.org/plugins/scm-landingpage-plugin/) - Shows a personal page for each user including all related information
+- [Content Search](https://scm-manager.org/plugins/scm-content-search-plugin/) - Enables search inside the files of your repositories
+
+## Blog Post Series
+
+We are aware that we are mostly developers and marketing seems not to be our greatest strength. SCM-Manager is getting bigger 
```

Comment:
```suggestion
The SCM-Manager is getting bigger 
```
Although this might be true and honest, I don't think that this sentence adds to the overall point of the message.